### ISSUE TITLE
News downloader: maintain multiple selectors

### ISSUE THREAD

**Issue by pideveloper** [koreader/koreader#11430]
* Device: Kobo Glo

#### Issue

I'm trying to optimize the process of downloading news to my ereader.  I was wondering if it is possible to select multiple css classes with the filter_element function?

**Comment by Frenzie** [comment]
At a glance I don't see why not, unless it doesn't handle commas. Have you tried it?

**Comment by pideveloper** [comment]
Thank you for your quick response. I tried it out the following ways but it
still seems to scrape the full webpage.

   - "artstyle__header";"artstyle__main--container"
   - "artstyle__header","artstyle__main--container"
   - artstyle__header,artstyle__main--container

I'm changing the settings on the e-reader directly.

Op do 1 feb 2024 om 09:02 schreef Frans de Jonge ***@***.***>:

**Comment by Frenzie** [comment]
You mean they do work if you stick to just the one? In that case it's not supported, though normally I'd expect something more like `.artstyle__header, .artstyle__main--container` since that doesn't sound like normal element names.

**Comment by Frenzie** [comment]
I agree in this case it probably makes more sense to take a table. Could you submit a PR?
